examples/roitberg/roitberg.py

```diff
@@ -17,8 +17,9 @@ def convert_species_to_atomic_nums(s):
 
 
 # replace with your own scratch directory
-data_dir = "/media/yutong/datablob/datasets"
-model_dir = "/media/yutong/datablob/models"
+data_dir = "/media/yutong/fast_datablob/datasets"
```

Comment:
this whole file should be in contrib vs examples.  examples should have relative paths along with the tools to download data, and is held to a higher standard for quality.

Reply:
This isn't for data-loading. This is for saving/caching featurized data/models - I don't think we have a consensus on how to do this yet. I could just leave this to empty or environment variable or default to /tmp/. I also want to keep this in examples so it's maintained.

Reply:
``` python
data_dir = deepchem.utils.get_data_dir() + "/fast_datablob/datasets"
```